index.js

```diff
@@ -82,11 +100,47 @@ cm.Item({
   }
 });
 
+// Vine context menu on Vine.co
+// only works from direct embed URLs structured like this: 
+// https://vine.co/v/[vineID] or https://vine.co/v/[vineID]/embed
+cm.Item({
+  label: "Send to mini player",
+  context: [
+  	cm.PredicateContext(checkLocation),
+  	cm.SelectorContext('[src*="mediasource:https://vine.co/"]')
```

Comment:
Does this prevent this from launching with regular vine links? https://vine.co/v/hW5WxZVjJYP